src/sources/util/unix_datagram.rs

```diff
@@ -78,19 +79,27 @@ async fn listen(
                     })
                 })?;
 
+                let span = info_span!("datagram");
+        let received_from = if !address.is_unnamed() {
+                    let path = address.as_pathname().map(|e| e.to_owned()).map(|path| {
+            span.record("peer_path", &field::debug(&path));
+            path
+                    });
```

Comment:
formatting looks off here...

Reply:
I agree, but it's the output of the rust formatter. I tried checking in the "correct" formatting, and the formatting Github action rejected the change lol

Reply:
Just guessing, but if you put a newline before the `.map` I think it tends to format stuff like this better.